### PR TITLE
update ERE readers

### PR DESCRIPTION
**Pull request by mssammon**
simplified use by providing coarser configuration.  
Readers debugged: now handle multiple ERE releases (which have different directory structures, file naming conventions, and markup specifications). 
Added multilingual ERE tests in external.

### CODE REVIEW

File: corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/ereReader/EREDocumentReader.java [hunk 1]
```diff
@@ -26,22 +28,34 @@
...
  * @author msammon
  */
 public class EREDocumentReader extends XmlDocumentReader {
 
-    /**
+        /**
      * tags in document files
      */
-    public static final String QUOTE = "quote";
+    public static final String QUOTE = "quote";;
```
Comment (danyaljj) [inline corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/ereReader/EREDocumentReader.java]: is it syntax error?

**Comment by danyaljj** [comment]
Some files are missing license header: `mvn license:format`

File: external/pom.xml [hunk 2]
```diff
@@ -23,6 +23,11 @@
             <version>1.9.7</version>
         </dependency>
         <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>wikipediaAPI-multilingual</artifactId>
+            <version>1.1.12</version>
+        </dependency>
+        <dependency>
```
Comment (danyaljj) [inline external/pom.xml]: a clarification question: what does this package include? Is it just mutlilingual tokenizers?

Comment (mssammon) [inline external/pom.xml]: mainly this is why I need it. I could extract the MultilingualTokenizer, I guess...

File: corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/ereReader/EREDocumentReader.java (hunk 1 above)
Comment (mssammon) [inline corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/ereReader/EREDocumentReader.java]: yes, Fixed now.

**Comment by danyaljj** [comment]
Tests passing. Feel free to merge.

File: corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/ereReader/EREMentionRelationReader.java [hunk 3]
```diff
@@ -203,7 +225,7 @@ public void readRelation(Node node, View view) throws XMLException {
     }
 
     @Override
-    public String getViewName() {
+    public String getNerViewName() {
```
Comment (mssammon) [inline corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/ereReader/EREMentionRelationReader.java]: agreed. Fixing.

**Pull request merged**